README.spanish.md

```diff
@@ -1017,23 +1066,23 @@ Todo lo de encima hubiera devuelto `false` si se hubiera usado `===`
 
 ## ![✔] 7.1. No bloquees el bucle de eventos
 
-**TL;DR:** Evita las tareas intensivas de la CPU, ya que bloquearán el bucle de eventos en su mayoría de un solo subproceso y las descargarán en un subproceso dedicado, proceso o incluso una tecnología diferente según el contexto.
+**TL;DR:** Evita las tareas intensivas de la CPU, ya que bloquearán el bucle de eventos en su mayoría de un solo subproceso y descargalas en un subproceso dedicado, proceso o incluso una tecnología diferente según el contexto.
```

Comment:
"las descargarán" === "**they** will download" which doesn't make sense
"descargalas" === "(**you**) download"
There's no proper "off load" for this case, but descargar is the most accurate